metricflow-semantics/metricflow_semantics/specs/where_filter/where_filter_dimension.py

```diff
@@ -151,5 +155,7 @@ def create(self, name: str, entity_path: Sequence[str] = ()) -> WhereFilterDimen
             rendered_spec_tracker=self._rendered_spec_tracker,
             element_name=structured_name.element_name,
             entity_links=tuple(EntityReference(entity_link_name.lower()) for entity_link_name in entity_path)
-            + structured_name.entity_links,
+            + tuple(
```

Comment:
this change is just to support case insensitivity right?

Reply:
nope, we switched from using `DunderedNameFormatter` to `StructuredLinkableSpecName` and that didn't have `entity_links: Tuple[EntityReference, ...]`, but only `entity_link_names: Tuple[str, ...]`. Now that I think about it, i'll just add a property to make `entity_links` work in `StructuredLinkableSpecName` so that we don't need this change

Reply:
Ooo ok sounds good!